tests/integration/find-test.js

```diff
@@ -1,27 +1,49 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
-import { find, first } from 'ember-native-dom-helpers/test-support/helpers';
+import { find, findAll } from 'ember-native-dom-helpers/test-support/helpers';
```

Comment:
Ok @rwjblue @cibernox I did put these into two tests, but I also separated out all the modules into individual files.